src/VirtualClient/VirtualClient.Actions/SysbenchOLTP/SysbenchOLTPClientExecutor.cs

```diff
@@ -289,7 +287,7 @@ private void CaptureMetrics(IProcessProxy process, EventContext telemetryContext
 
                         this.Logger.LogMetrics(
                             toolName: "MySQL-Sysbench",
-                            scenarioName: "OLTP " + this.Scenario,
+                            scenarioName: "OLTP " + this.MetricScenario ?? this.Scenario,
```

Comment:
No need to have "OLTP " +

Just scenarioName: this.MetricScenario ?? this.Scenario